docs/3.0.0-beta.x/guides/custom-data-response.md

```diff
@@ -0,0 +1,117 @@
+# Custom data response
+
+In this guide we will see how you can customize the data of your APIs responnses.
+
+## Introduction
+
+To be able to update the default data response you have first to understand how it works.
+
+When you create a content type, it generate an API with the following list of [endpoints](../content-api/endpoint).
+
+Each of these endpoint trigger a controller action. Here is the list of [controller actions](../concepts/controller.md) that exist by default when a content type is created.
+
+If you check the controller file of your generated API `./api/{content-type}/controller/{Content-Type}.js`, you will see an empty file. It's because all the default logic is managed by Strapi. But we allow your override these functions with your own logic.
```

Comment:
in your generated API. It is  because...

But you can override these actions with your own code.